src/DynamoCoreWpf/Views/PackageManager/Controls/SearchBoxControl.xaml

```diff
@@ -116,8 +116,7 @@
                             HorizontalAlignment="Stretch"
                             IsEnabled="{Binding RelativeSource={RelativeSource AncestorType=UserControl}, Path=IsEnabled}"
                             TextChanged="SearchTextBox_OnTextChanged"
-                            IsKeyboardFocusWithinChanged="SearchTextBox_OnKeyboardFocusWithinChanged"
-                            Text="{Binding SearchText, UpdateSourceTrigger=PropertyChanged}"></TextBox>
```

Comment:
Removed the direct binding. We are already setting the text via `SearchTextBox_OnKeyboardFocusWithinChanged`